tests/test_user_util.py

```diff
@@ -43,7 +43,7 @@
 def test_cli_with_no_options():
     runner = CliRunner()
     result_w_no_options = runner.invoke(cli.retire_user)
-    assert result_w_no_options.exit_code == -1
+    assert result_w_no_options.exit_code != 0
```

Comment:
why exit code is not returning -1 now ?

Reply:
It is because of these change in the Click package https://github.com/pallets/click/pull/739/commits/a01500bf8f225ce2f133dd2fc238bcebef9fa5c4.
This test purpose is to validate that the exit code will not be 0 (successful) when any options are provided so this change is not modifying the test goal.